### PR TITLE
lock omniauth-oauth2 version until there is an upstream fix

### PR DESCRIPTION
**Pull request by pkarman**
omniauth-oauth2 1.4.0 breaks myusa integration, via https://github.com/intridea/omniauth-oauth2/commit/26152673224aca5c3e918bcc83075dbb0659717f

### CODE REVIEW

**Comment by afeld** [comment]
Worked locally. QA-ing on dev.

File: Gemfile [hunk 1]
```diff
@@ -29,6 +29,7 @@ gem 'jquery-rails'
 gem 'jquery-turbolinks'
 gem 'mandrill-rails'
 gem 'newrelic_rpm'
+gem 'omniauth-oauth2', '1.3.1'
```
Comment (afeld) [inline Gemfile]: Mind adding a note about the reasoning?

Comment (afeld) [inline Gemfile]: Also, is there a relevant issue we can link to?

**Comment by afeld** [comment]
Worked for me! Just the one minor note above – go ahead and merge when done.

**Comment by pkarman** [comment]
there are several comments on the commit link in the description, including this one https://github.com/intridea/omniauth-oauth2/commit/26152673224aca5c3e918bcc83075dbb0659717f#commitcomment-14379780

**Comment by pkarman** [comment]
also filed as https://github.com/intridea/omniauth-oauth2/issues/81 with the same recommended fix we used.

**Pull request merged**